modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/js/store/withStore.es.js

```diff
@@ -52,6 +52,7 @@ const _handleFieldEdited = function(properties) {
 const _handleFieldBlurred = function(properties) {
 	const {fieldInstance} = properties;
 	const {pages} = this;
+	const now = new Date();
```

Comment:
@interaminense, can we change this variable name to `dateNow` or `currentDate`? I think it'll be better for the developers understanding. What do you think about it?

Reply:
Yes, @natocesarrego! Done: https://github.com/natocesarrego/liferay-portal/pull/177/commits/c961672db6885e1156b018b5a21cc12abec973d5